ush/python/pygfs/task/aero_analysis.py

```
@@ -101,17 +101,18 @@ def initialize(self) -> None:
         logger.debug(f"Observation files:\n{pformat(obs_dict)}")
 
         # # stage bias corrections
-        # logger.info(f"Staging list of bias correction files")
-        # bias_dict = self.jedi_dict['aeroanlvar'].render_jcb(self.task_config, 'aero_bias_staging')
-        # if bias_dict['copy'] is None:
-        #     logger.info(f"No bias correction files to stage")
-        # else:
-        #     bias_dict['copy'] = Jedi.remove_redundant(bias_dict['copy'])
-        #     FileHandler(bias_dict).sync()
-        #     logger.debug(f"Bias correction files:\n{pformat(bias_dict)}")
-
-        #     # extract bias corrections
-        #     Jedi.extract_tar_from_filehandler_dict(bias_dict)
+        logger.info(f"Staging list of bias correction files")
+        bias_dict = self.jedi_dict['aeroanlvar'].render_jcb(self.task_config, 'aero_bias_staging')
+
+        if bias_dict['copy'] is None:
+            logger.info(f"No bias correction files to stage")
```

Will the JEDI code fail if it looks for BC files but they are missing? If so, should we raise an exception here?